.eslintrc.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
 		'computed-property-spacing': [ 1, 'always' ],
 		'func-call-spacing': 1,
 		indent: [ 1, 'tab', { SwitchCase: 1 } ],
+		'jsx-a11y/accessible-emoji': 1,
+		'jsx-a11y/click-events-have-key-events': 1,
+		'jsx-a11y/heading-has-content': 1,
+		'jsx-a11y/href-no-hash': 1,
+		'jsx-a11y/html-has-lang': 1,
+		'jsx-a11y/iframe-has-title': 1,
+		'jsx-a11y/img-has-alt': 1,
+		'jsx-a11y/label-has-for': 1,
```

Comment:
You're right — but I think it should still be included, just as a reminder to _not_ do your first example... unless you think that would just confuse people? Maybe it would. Hm...

There's also discussion of updating this rule to allow the nested version to pass: https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/51 — maybe we disable this one here in calypso, but leave it in the parent config, and if that update happens, we can bump this back up to warning.